doc/src/manual/mathematical-operations.md

```diff
@@ -9,18 +9,18 @@ collection of standard mathematical functions.
 The following [arithmetic operators](https://en.wikipedia.org/wiki/Arithmetic#Arithmetic_operations)
 are supported on all primitive numeric types:
 
-| Expression | Name           | Description                            |
-|:---------- |:-------------- |:-------------------------------------- |
-| `+x`       | unary plus     | the identity operation                 |
-| `-x`       | unary minus    | maps values to their additive inverses |
-| `x + y`    | binary plus    | performs addition                      |
-| `x - y`    | binary minus   | performs subtraction                   |
-| `x * y`    | times          | performs multiplication                |
-| `x / y`    | divide         | performs division                      |
-| `x ÷ y`    | integer divide | x / y, truncated to an integer         |
-| `x \ y`    | inverse divide | equivalent to `y / x`                  |
-| `x ^ y`    | power          | raises `x` to the `y`th power          |
-| `x % y`    | remainder      | equivalent to `rem(x,y)`               |
+| Expression | Name           | Description                                              |
+|:---------- |:-------------- |:---------------------------------------------------------|
+| `+x`       | unary plus     | the identity operation                                   |
+| `-x`       | unary minus    | maps values to their additive inverses                   |
+| `x + y`    | binary plus    | performs addition                                        |
+| `x - y`    | binary minus   | performs subtraction                                     |
+| `x * y`    | times          | performs multiplication                                  |
+| `x / y`    | divide         | performs division                                        |
+| `x ÷ y`    | integer divide | x / y, truncated to an integer, ÷ is typed by `\div<tab>`|
```

Comment:
Rather than trying to squeeze it into the table, how about putting this note after the table?

Reply:
That makes sense. I'll write a sentence below the table. Another option could be to write a footnote with `[^1]`